lnd_services.go

```diff
@@ -127,25 +126,25 @@ func NewLndServices(cfg *LndServicesConfig) (*GrpcLndServices, error) {
 	macaroonDir := cfg.MacaroonDir
 	if macaroonDir == "" {
 		switch cfg.Network {
-		case "testnet":
+		case NetworkTestnet:
```

Comment:
Couldn't this whole switch statement now be removed as `cfg.Network` is now guarded against being an unknown network?

Reply:
It isn't guarded any more now, because we move to string